libs/langchain/langchain/document_loaders/directory.py

```diff
@@ -39,6 +41,9 @@ def __init__(
         show_progress: bool = False,
         use_multithreading: bool = False,
         max_concurrency: int = 4,
+        sample_size: Union[int, None] = None,
```

Comment:
Could you add `*` here

```suggestion
        *,
        sample_size: Union[int, None] = None,
```

Reply:
Will do. thanks for the review.